qiling/os/posix/syscall/wait.py

```diff
@@ -10,12 +10,19 @@
 from qiling.os.filestruct import *
 from qiling.os.posix.const_mapping import *
 from qiling.exception import *
+from qiling.os.macos.const import ECHILD
```

Comment:
Macos is somewhat similar to posix and I did not find such constants for unix/posix in qiling. I checked the linux kernel source code, where it is currently 10 and  `qiling.os.macos.const.ECHILD` is 10 as well. If there are constants available for posix please let me know. Declaring the ECHILD constant inside the wait.py file felt like a bad idea and I didn't know where it would fit better.

Reply:
If this ECHILD is needs to use by both MacOS and Linux. This value must be in posix and not import mac from macos.

Reply:
I added ECHILD to posix/const.py and import from there.